manager_f1tenth.py

```diff
@@ -66,7 +83,7 @@ def exec_container():
     os.system("docker exec -it f1tenth_gym_ros-sim-1 /bin/bash")
```

Comment:
Modify f1tenth_gym_ros-sim-1 to reflect changes made from docker-compose -> docker compose.